### DISCUSSION TITLE
best way to route within subfolder.

### DISCUSSION THREAD

**Discussion by meodai** [koajs/koa#1731] [Q&A] [answered]
I am running different API's on the same URL. (With separate codebases)

for now my "color-names" api is running here: https://api.color.pizza/v1/

In the future I want to have: 
```
https://api.color.pizza/names/v1/  <- current codebase (vanilla JS)
& 
https://api.color.pizza/palettes/v1/ <- new API built around Koa
```

I want to avoid having to add `palettes/v1/` as a prefix to every route. Is it possible to solve that nicely using the "host" feature?
Also how would I still be able to be using it on localhost? 

```js
const router = new Router({
  host: /^(.*\.)?color\.pizza\/palettes\/v1/
})
```

**Comment by jonathanong** [comment, marked as answer]
you can use `koa-mount` for this
